### ISSUE TITLE
Merge into CourtListener Proper; Productize

### ISSUE THREAD

**Issue by mlissner** [anseljh/casebot#10]
OK, so I really like casebot, and really wish it worked better. 

With all your copious time, would you be interested in making it a part of CourtListener proper, and then advertising it properly in Slack's store? If it were part of CourtListener proper, it'd certainly be more reliable than...under your desk?

Roadmap might be as short as:

1. Fix query injection #9 
2. Copy/paste stuff to CourtListener app & update to not use the API (or maybe it still does, over localhost, IDK)
3. Update the Slack app store
4. Blog post?

**Comment by anseljh** [comment]
Good idea! I looked into turning it into a proper Slack app a while back, and would love to do that.
